tests/integration/components/instructor-groups/list-test.js

```diff
@@ -44,7 +47,11 @@ module('Integration | Component | instructor-groups/list', function (hooks) {
   });
 
   test('it renders empty', async function (assert) {
-    await render(hbs`<InstructorGroups::List @programs={{(array)}} />`);
+    await render(hbs`<InstructorGroups::List
+      @programs={{(array)}}
+      @sortBy="title"
+      @setSortBy={{this.sortBy}}
```

Comment:
```suggestion
      @setSortBy={{(noop)}}
```

I don't think `this.sortBy` exists.

Reply:
fixed :heavy_check_mark: